src/app/shared/footer/footer.ts

```diff
@@ -1,5 +1,7 @@
 import {Component, NgModule} from '@angular/core';
 import {materialVersion} from '../version/version';
+import { RouterModule } from '@angular/router';
```

Comment:
Please remove the spaces surrounding the import.

```suggestion
import {RouterModule} from '@angular/router';
```